src/Raven.Server/Config/Categories/ClusterConfiguration.cs

```diff
@@ -106,5 +106,11 @@ public class ClusterConfiguration : ConfigurationCategory
         [DefaultValue(2048)]
         [ConfigurationEntry("Cluster.LogHistoryMaxEntries", ConfigurationEntryScope.ServerWideOnly)]
         public int LogHistoryMaxEntries { get; set; }
+
+        [Description("Time (in seconds) between expired compare exchange cleanup")]
+        [DefaultValue(5)]
+        [TimeUnit(TimeUnit.Seconds)]
+        [ConfigurationEntry("Cluster.ExpiredCompareExchangeCleanupIntervalInSec", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
`Cluster.CompareExchangeExpiredDeleteFrequencyInSec` to match the ExpirationConfiguration